site/src/sphinx/auth.rst

```diff
@@ -0,0 +1,133 @@
+.. _`Apache Shiro`: https://shiro.apache.org/
+
+.. _auth:
+
+Authentication and Access Control
+=================================
+
+*Central Dogma* provides the features for authentication and access control.
+
+Authentication
+--------------
+
+For authentication, `Apache Shiro`_ is used as its authentication layer and ``conf/shiro.ini`` file is used
+as its security configuration. Please follow the steps below to turn on the security system of *Central Dogma*.
+
+First, enable the security system by configuring ``conf/dogma.json``. Specify login IDs of the administrators there, too::
+
+    {
+        ...
+        "securityEnabled": true,
+        "administrators": [
+            "system-admin", "dogma-admin"
+        ]
+        ...
+    }
+
+Next, configure ``conf/shiro.ini`` based on your authentication system. A local database system of `Apache Shiro`_
+is used here to show you a simple example::
+
+    [users]
+    system-admin = system-admin-secret
+    dogma-admin = dogma-admin-secret
+
+.. note::
+
+    `Apache Shiro`_ supports RDBMS or LDAP based security system as well. You can find the example configuration
+    files under the ``conf/`` directory in the distribution.
+
+That's all. You are ready to use the security system of *Central Dogma*.
+
+Access Control
+--------------
+
+One of the way of configuring the access control system for *Central Dogma* is setting a project with a web UI.
+To open the web UI, you may access ``http://{your-central-dogma-domain-or-ip}:36462`` by your web browser.
+You may configure a project with HTTP APIs, but we recommend the web UI because it is easier and simple.
+
+Everyone who is logged in is able to create a new project, and he or she would be an owner of the project.
+If you have a right to configure a project, in other words, if you are an owner of the project,
+you can access the configuration UI of the project by clicking the pencil icon which is shown on the right
+of the project name.
+
+.. image:: _images/auth_1.png
+
+You can see the configuration UI for a project, in the below example it is for the project ``sample``.
+With this UI, you can add a user or token as a member of the project. You can also remove the member or token from the project.
```

Comment:
- With this UI -> In this page
- a user or a token
- the member of token -> them